scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJob.java

```diff
@@ -98,12 +98,17 @@
     protected Map<TaskId, InternalTask> tasks = new ConcurrentHashMap<>();
 
     /**
-     * Informations (that can be modified) about job execution
+     * Information (that can be modified) about job execution
      */
     protected JobInfoImpl jobInfo = new JobInfoImpl();
 
     /**
-     * Job descriptor for dependences management
+     * Merged map of job results
+     */
+    private Map<String, Serializable> JobMap = new ConcurrentHashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)